2-mandatory/8-password-validator.js

```diff
@@ -23,11 +23,14 @@ PasswordValidationResult=  [false, false, false, false, true]
 
 */
 
-function validatePasswords(passwords) {}
+     
+
+
+
 
 // Returns true if string contains at least one uppercase letter.
 function containsUppercaseLetter(string) {
-  return /[A-Z]/.test(string);
```

Comment:
we have not covered Regex much in class, but you should try to copy the example here. For example, a lower case test would be `/[a-z]/.test()` and a number would be `/[0-9]/.test()`